common/taskana-common/src/main/java/pro/taskana/common/internal/test/InitOracleSchema.java

```diff
@@ -0,0 +1,70 @@
+package pro.taskana.common.internal.test;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.Statement;
+import javax.sql.DataSource;
+
+import pro.taskana.common.api.exceptions.SystemException;
+
+/**
+ * Helper Class to create a proper ORACLE SCHEMA
+ *
+ * <p>This class is only used in TEST classes. DON'T USE IT IN PRODUCTION CODE.
+ */
+public final class InitOracleSchema {
+
+  public static void initOracleSchema(DataSource dataSource, String schemaName)
+      throws SystemException {
+    try (Connection connection = dataSource.getConnection()) {
+      Connection conn =
+          DriverManager.getConnection(connection.getMetaData().getURL(), "SYSTEM", "testPassword");
+      try (Statement stmt = conn.createStatement()) {
+        stmt.execute("GRANT ALL PRIVILEGES TO TEST_USER");
+
+        if ("TASKANA".equals(schemaName)) {
+          try {
+            stmt.execute("DROP USER " + schemaName + " CASCADE");
+            stmt.execute(
+                "DROP TABLESPACE "
+                    + schemaName
+                    + "_TMP INCLUDING CONTENTS AND DATAFILES CASCADE CONSTRAINTS");
+            stmt.execute(
+                "DROP TABLESPACE "
+                    + schemaName
+                    + " INCLUDING CONTENTS AND DATAFILES CASCADE CONSTRAINTS");
+          } catch (Throwable e) {
+            // IGNORE exceptions from dropping
+          }
```

Comment:
when duplicating up this file:
In the test-api: remove this
for the common-test module: always do this